ani-cli

```diff
@@ -453,15 +453,17 @@ open_episode () {
 
 play_episode () {
 	# Build command
-	set -- "$player_fn" "$video_url"
+trackma_title="$(printf '%s' "$anime_id" | tr '-' ' ' | awk '{for(i=1;i<=NF;i++){ $i=toupper(substr($i,1,1)) substr($i,2) }}1')"
 	case "$player_fn" in
 		vlc)
 			[ ! "$auto_play" -eq 0 ] && set -- "$@" "--play-and-exit"
-			set -- "$@" --http-referrer="$dpage_link"
+			set -- "$player_fn" "$video_url"  --http-referrer="$dpage_link"
+			;;
+		syncplay|/Applications/Syncplay.app/Contents/MacOS/syncplay)
+			nohup "$player_fn" "$video_url" -- --referrer="$dpage_link" --force-media-title="$trackma_title $episode" > /dev/null 2>&1 &
```

Comment:
I'm actually a bit worried about this line's compatibility with auto-play mode (whenever you specify an episode range). It's fine to exclude that option from syncplay if it doesn't make much sense, what I'm worried about is the program ending up in an undefined state. I'd rather have it play from line 471 or 474

Reply:
at the moment it's not compatible with auto-play mode, but ideally i think it should be imo. i'll look into this tomorrow, ty for the suggestion

Reply:
nvm, it was an easy fix, i just decided to use both my eyes and brain at the same time for once

Reply:
everything working as expected now, don't know why i complicated things in the beginning, sorry for that